package.json

```diff
@@ -21,17 +21,10 @@
   "types": "src/hyparquet.d.ts",
   "scripts": {
     "coverage": "vitest run --coverage --coverage.include=src",
-    "demo": "http-server -o",
-    "demo:build": "rollup -c",
     "lint": "eslint .",
     "test": "vitest run"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "28.0.1",
-    "@rollup/plugin-node-resolve": "15.3.0",
-    "@rollup/plugin-replace": "6.0.1",
-    "@rollup/plugin-terser": "0.4.4",
-    "@rollup/plugin-typescript": "12.1.1",
     "@types/node": "22.8.6",
     "@types/react": "18.3.12",
```

Comment:
I think we can remove react, react-dom and types too, right?

Reply:
sure

Reply:
fixed